packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
We should do a version bump and [README update](https://github.com/flutter/plugins/blob/main/packages/in_app_purchase/in_app_purchase/README.md?plain=1#L10) here, as with ios_platform_images.

(It's arguable in the federated world what "supported version" actually means, since technically you could use 3.1.1 or whatever with iOS 9 and an older version of Flutter and the resolver would just give you an older version of the implementation package, but I think having the current implementation's version in the main package is the least confusing.)

Reply:
You're right, not sure how I missed this chart in the README:
https://github.com/flutter/plugins/blob/94a54bf3acdf5b0e2c935363e7b2bddcca549334/packages/in_app_purchase/in_app_purchase/README.md?plain=1#L10

Will add a patch bump.